getting-started/events.md

```diff
@@ -0,0 +1,268 @@
+    "label": "Your email",
+    "required": true,
+    "value": "jan.kowalski@gmail.com"
+  }
+```
+
+`radio`:
+
+```js
+  {
+    "type": "radio",
+    "name": "purpose",
+    "label": "Chat purpose",
+    "required": true,
+    "options": [{
+      "label": "Support",
+      "value": "support",
+      "checked": true
+    },
+    {
+      "label": "Sale",
+      "value": "sale",
+      "checked": false
+    }],
+  }
+```
+
+`checkbox`:
+
+```js
+  {
+    "type": "checkbox",
+    "name": "industry",
+    "label": "Company industry",
+    "required": true,
+    "options": [{
+      "label": "automotive",
+      "value": "automotive",
+      "checked": true
+    }, {
+      "label": "IT",
+      "value": "it",
+      "checked": true
+    }],
+  }
+```
+
+`select`:
+
+```js
+{
+  "type": "select",
+  "name": "country",
+  "label": "Country",
+  "required": true,
+  "options": [{
+    "label": "USA",
+    "value": "usa",
+    "checked": false
+  }, {
+    "label": "Poland",
+    "value": "pl",
+    "checked": true
+  }, {
+    "label": "Poland",
+    "value": "pl",
+    "checked": false
+  }]
+}
+```
+
+
+### System message
+
+System message is event generated by server. It does not have "author_id", "custom_id" and "properties" fields.
+
+```js
+{
+  "id": "0affb00a-82d6-4e07-ae61-56ba5c36f743",
+  "order": 1,
+  "type": "system_message",
+  "timestamp": 1473433500,
+  "text": "Mike joined the chat",
+  "system_message_type": "agent_joined"
+}
+```
+
+System messages can be triggered by an user action within chat or by router (more about router messages in future Router section). Each action triggers system message with different `system_message_type` and has different `text` message:
+
+| system_message_type | text                             |
+|---------------------|----------------------------------|
+| `agent_joined`      | `<agent_name> joined the chat`   |
+| `agent_left`        | `<agent_name> left the chat`     |
+| `manual_archived`   | `<agent_name> archived the chat` |
+
+
+### File
+
+File event indicates that a file was uploaded. It can be only generated by server when [send file](../../customer-api/client-server#send-file) customer API method is called.
+
+```js
+{
+  "id": "0affb00a-82d6-4e07-ae61-56ba5c36f743",
+  "custom_id": "12312.301231238591134",
+  "order": 1,
+  "type": "file",
+  "author_id": "b7eff798-f8df-4364-8059-649c35c9ed0c",
+  "timestamp": 1473433500,
+  "properties": {
+    // "Properties" object
+  }
+  "name": "image25.png",
+  "url": "https://domain.com/asdsfdsf.png",
+  "content_type": "image/png",
+  "size": 123444,
+  "width": 640,
+  "height": 480
+}
+```
+
+The `size` field means file size in bytes. It is limited to 10 MB. <br>
+The `content-type` field can contain any [MIME media type](https://en.wikipedia.org/wiki/Media_type). <br>
+Fields `width` and `height` are present only for `"image/png"`, `"image/gif"` and `"image/jpg"` content-types. There is no limitations for width and height.
```

Comment:
Fields `width` and `height` are present only for `"image/png"`, `"image/gif"` and `"image/jpg"` content-types. There is no limitations for width and height. -> `Width` and `height` fields are present only for `"image/png"`, `"image/gif"` and `"image/jpg"` content-types. There are no limitations for width and height.